src/components/clusterConfiguration/BasicNetworkFields.tsx

```diff
@@ -1,13 +1,14 @@
 import React from 'react';
 import { HostSubnets, ClusterConfigurationValues } from '../../types/clusters';
-import { InputField, SelectField } from '../ui/formik';
+import { InputField, SelectField, SwitchField } from '../ui/formik';
 import { useFormikContext } from 'formik';
 
 type BasicNetworkFieldsProps = {
   hostSubnets: HostSubnets;
+  isAdvanced: boolean;
 };
 
-const BasicNetworkFields: React.FC<BasicNetworkFieldsProps> = ({ hostSubnets }) => {
+const BasicNetworkFields: React.FC<BasicNetworkFieldsProps> = ({ hostSubnets, isAdvanced }) => {
```

Comment:
yes, i know this is ugly. Passing `isAdvanced` to `BasicNetworkFields` - but we need this since we want to place the DHCP switch between basic fields. I could refactor the code but once the DHCP will be turned off we will want to go back to Basic and AdvancedNetworkingFields as we have now

Reply:
I think DHCP should be considered basic, because at effects the behavior of basic networking fields

Reply:
@ori-amizur what do you mean by it effects the behavior...
DHCP is not the default hence it's under advanced options..

Reply:
We will add it to Basic once its on by default in backend. This is just a temporary measure

Reply:
So this https://github.com/mareklibra/facet-lib/pull/161/files#diff-f936227366e52be8b7bf47d374fbc45cR42-R59 will move to Advanced.

Reply:
@eranco74 If you set DHCP, then vips filling should be disabled.  Vips filling is basic.
In addition, validations will behave differently.

Reply:
1. It is an experimental feature, not advanced.  If it's too complicated now to make a new section, then advanced is the best option.
2. Can we somehow indicate to the user that this option exists?  For example near where the user enters the VIPs write something like "If you don't have available IP addresses you may try the experimental option of automatically allocating them".

Reply:
Also, machine-cidr selection should have different functionality, since only in DHCP mode, the UI should include it in the UpdateCluster request

Reply:
@avishayt why do you say it is experimental?

@ori-amizur Once DHCP is default, VIPs setting is not basic anymore, its just an output field.
@rawagner Notice that when DHCP is the default, the user must still see the values for the VIPs, so it cannot be hidden in the advanced section. I think that when DHCP is enabled, the VIPs field should turn into output fields

Reply:
It's not production-grade from what I understand because there are known cases where the lease will expire and then the cluster is hosed.

Reply:
@avishayt Can you indicate what cases?